static/js/components/LearnerSearch.js

```diff
@@ -328,7 +328,13 @@ export default class LearnerSearch extends SearchkitComponent {
 
   render () {
     const { currentProgramEnrollment } = this.props;
+    // Remove any filters that are still applied by searchkit, but don't appear in the querystring
+    let hasFiltersOtherThanSelectedProgram = _.get(this, 'searchkit.query.index.filters.length', 0) > 1;
+    if (window.location && R.isEmpty(window.location.search) && hasFiltersOtherThanSelectedProgram) {
+      this.searchkit.getQueryAccessor().keepOnlyQueryState();
+    }
 
+    // Render the searchkit component
```

Comment:
@amir-qayyum-khan - This looks good, but I had a couple reasons for suggesting that we put this code in `static/js/components/search/WithSearchkitManager.js` rather than `static/js/components/LearnerSearch.js`

1. `this.searchkit.getQueryAccessor().keepOnlyQueryState()` changes the state of the searchkit component (see [this function](https://github.com/searchkit/searchkit/blob/f59f28398928c47ef0ec3b25706c224b008766b5/src/core/accessors/BaseQueryAccessor.ts#L15)). In cases where this state change is needed, I believe everything above the `LearnerSearch` component will already be rendered, and the state change will cause a re-render. I *think* that changing the state before `SearchkitProvider` is rendered with the `searchkit` prop will avoid that re-render.
1. To me it just makes a little more sense to make this change in the same place that we instantiate the `SearchkitManager` than to make it in a specific component which happens to be the only component that leverages searchkit.

@aliceriot Do you have any opinions on the 2 points above (particularly the first)?